src/js/controllers/addresses.js

```diff
@@ -63,6 +65,8 @@ angular.module('copayApp.controllers').controller('addressesController', functio
         };
         $scope.allAddresses = $scope.noBalance.concat(withBalance);
 
+        console.log('withBalance', withBalance)
+
         cachedWallet = $scope.wallet.id;
```

Comment:
Do we need the console.logs in this file ? 👾

Reply:
Removed